<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>employee_management_window</class>
 <widget class="QWidget" name="employee_management_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>501</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#55aa00;&quot;&gt;Employee Management&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="employee_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>1001</width>
     <height>501</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>First Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Last Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>City</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>State</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Zip Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SSN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Withholdings</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Salary</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_employee_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>610</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_employee_button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>610</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Employee</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
